src/components/Form.spec.js

```diff
@@ -0,0 +1,34 @@
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { MemoryRouter } from 'react-router-dom';
+import Form from './Form';
+
+describe('Form', () => {
+  it('renders a form with three inputs and a submit button, does not submit wenn the input is empty or wenn there are only whitespaces and it has a title', () => {
```

Comment:
A little typo = "does not submit WHEN..." :)

Reply:
thank you! :)